docs/investigation/2022-06-14-policies.md

```diff
@@ -0,0 +1,379 @@
+# Polices
```

Comment:
If all these 3 documents are very related to each other (it is about improving configuration) and they was created during the same epic, WDYT about aggregating them in the `configuration` directory?